akka-bench-jmh/src/main/scala/akka/actor/ForkJoinActorBenchmark.scala

```diff
@@ -72,6 +72,56 @@ class ForkJoinActorBenchmark {
 
   @Benchmark
   @Measurement(timeUnit = TimeUnit.MILLISECONDS)
+  @OperationsPerInvocation(totalMessagesLessThanCores)
+  def pingPongLessActorsThanCores(): Unit = {
+    val pingPongs =
+      for {
+        i <- 1 to lessThanCoresActorPairs
+      } yield {
+        val ping = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        val pong = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        (ping, pong)
+      }
+
+    pingPongs.foreach { case (ping, pong) => ping.tell(message, pong) }
+
+    pingPongs.foreach {
+      case (ping, pong) =>
+        val p = TestProbe()
+        p.watch(ping)
+        p.expectTerminated(ping, timeout)
+        p.watch(pong)
+        p.expectTerminated(pong, timeout)
+    }
+  }
+
+  @Benchmark
+  @Measurement(timeUnit = TimeUnit.MILLISECONDS)
+  @OperationsPerInvocation(totalMessagesMoreThanCores)
+  def pingPongMoreActorsThanCores(): Unit = {
+    val pingPongs =
+      for {
+        i <- 1 to moreThanCoresActorPairs
+      } yield {
+        val ping = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        val pong = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        (ping, pong)
+      }
+
+    pingPongs.foreach { case (ping, pong) => ping.tell(message, pong) }
```

Comment:
This one I didn't do anything about, or, the bench is parameterized on throughput and I added 10 to 1 and 5 which was already there. I'll try out sending more messages and see if anything changes.